packages/storybook/src/nimble/table/table.mdx

```diff
@@ -198,4 +198,38 @@ following behavior:
   range of rows to the existing selection.
     - Note: On macOS, use the `Command` key (`⌘`) instead of `CTRL`.
 
+## Blazor usage
+
+The `NimbleTable` requires that its data be set via the `SetDataAsync` method.
+The appropriate place to call this method is either in the `OnAfterRenderAsync`
+override of the hosting component or after that method has been called for the
+first time.
+
+As the `NimbleTable` is generic a client must supply its generic type in the
+markup using the `TData` property syntax. The following code represents a
+typical usage of the `NimbleTable`:
+
+```cs
+<NimbleTable TData="MyRecordType" @ref="_table">
+@code {
+    private NimbleTable<MyRecordType>? _table;
+    private IEnumerable<MyRecordType> TableData { get; set; } = Enumerable.Empty<MyRecordType>();
+    ...
+    public override async Task OnAfterRenderAsync(bool firstRender)
+    {
+        await base.OnAfterRenderAsync(firstRender);
+        await _table.SetDataAsync(TableData); // populate TableData before here
+    }
+
+    public class MyRecordType
+    {
+        ...
+    }
+}
+```
```

Comment:
I don't have a preference, but (if possible) I think we should choose either the separate razor / code sections like we did in dialog or settle on the approach shown here of using a code section inside the razor (assuming it's possible to keep them consistent, maybe model binding can't be done from inline `@code`? not sure).

Reply:
I've separated it into two sections.